src/main/java/org/fogbowcloud/manager/core/rest/ExceptionHandlerController.java

```diff
@@ -0,0 +1,43 @@
+package org.fogbowcloud.manager.core.rest;
+
+import org.fogbowcloud.manager.core.exceptions.TokenCreationException;
+import org.fogbowcloud.manager.core.exceptions.UnauthorizedException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
```

Comment:
out of curiosity (I know nothing about rest framework and stuff),  what is the purpose of this class @ordansantos?

Reply:
It maps an exception to an http response.